cmd/email-provider/server/operations.go

```
@@ -16,7 +20,69 @@ func NewServer() *Server {
+	}
+	p.AddTos(tos...)
+	p.Subject = os.Getenv("EMAIL_SUBJECT")
+	m.AddPersonalizations(p)
+
+	htmlContent := fmt.Sprintf(
+		`
+		 <strong> First name: </strong> %s <br> 
+		 <strong> Last name: </strong> %s <br>
+		 <strong> Email: </strong> %s <br> 
+		 <strong> Phone: </strong> %s <br><br> 
+		 <strong> Request details: </strong> %s <br> 
+		 <strong> Request service: </strong> %s
+		`,
+		message.Sender.FirstName, message.Sender.LastName, message.Sender.Email, message.Sender.PhoneNumber, message.RequestDetails, message.RequestService)
+	c := mail.NewContent("text/html", htmlContent)
+	m.AddContent(c)
+
+	mailSettings := mail.NewMailSettings()
+	bypassBounceManagement := mail.NewSetting(true)
+	mailSettings.SetBypassBounceManagement(bypassBounceManagement)
+	bypassSpamManagement := mail.NewSetting(true)
+	mailSettings.SetBypassSpamManagement(bypassSpamManagement)
+	bypassUnsubscribeManagement := mail.NewSetting(true)
+	mailSettings.SetBypassUnsubscribeManagement(bypassUnsubscribeManagement)
+
+	footerSetting := mail.NewFooterSetting()
+	footerSetting.SetText("footer")
+	footerSetting.SetEnable(true)
+	footerSetting.SetHTML("<html><body><br><br>Club CEDILLE, ETS</body></html>")
+	mailSettings.SetFooter(footerSetting)
+
+	spamCheckSetting := mail.NewSpamCheckSetting()
+	spamCheckSetting.SetEnable(true)
+	spamCheckSetting.SetSpamThreshold(1)
+	spamCheckSetting.SetPostToURL("https://spamcatcher.sendgrid.com")
+	mailSettings.SetSpamCheckSettings(spamCheckSetting)
+	m.SetMailSettings(mailSettings)
+
+	replyToEmail := mail.NewEmail(os.Getenv("EMAIL_FROM_NAME"), os.Getenv("EMAIL_FROM_ADDRESS"))
+	m.SetReplyTo(replyToEmail)
+
+	request := sendgrid.GetRequest(os.Getenv("EMAIL_SENDGRID_API_KEY"), "/v3/mail/send", "https://api.sendgrid.com")
+	request.Method = "POST"
+	var Body = mail.GetRequestBody(m)
+	request.Body = Body
+	response, err := sendgrid.API(request)
+	if err != nil {
+		log.Println(err.Error())
```

should return the error rather than logging it